impeller/entity/shaders/blending/advanced_blend.glsl

```diff
@@ -43,8 +43,7 @@ void main() {
                        kTileModeDecal                 // tile mode
                        ));
 
-  vec4 blended = vec4(Blend(dst.rgb, src.rgb), 1) * dst.a;
+  vec4 blended = vec4(Blend(f16vec3(dst.rgb), f16vec3(src.rgb)), 1) * dst.a;
```

Comment:
This is the only place we need to do a conversion from float to half, so assuming the half-based logic is more performant it seems like a reasonable trade-off.